.github/CODEOWNERS

```diff
@@ -255,6 +255,7 @@
 /core/controllers/review_tests*.py @sophiewu6
 /core/domain/question*.py @aks681
 /core/storage/question/ @aks681
+/core/templates/dev/head/components/concept-card/ @sophiewu6
```

Comment:
Hey @sophiewu6 , can you add me as the owner as well for this directory?

Reply:
Done.